types/echarts/index.d.ts

```diff
@@ -9,6 +9,8 @@ declare namespace ECharts {
         renderer?: string
     }):ECharts;
 
+    const graphic: any;
```

Comment:
👍 

FWIW, `LinearGradient` isn't included in the document but used in the official [demo](http://echarts.baidu.com/demo.html#bar-gradient).

```
import * as EChart from 'echarts';

new (<any>EChart.graphic).LinearGradient
```